src/main/java/Duke.java

```diff
@@ -1,10 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
 public class Duke {
+
+    private static void printFileContents(String filePath, ArrayList<Task> lst) throws FileNotFoundException {
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        while (s.hasNext()) {
+            System.out.println(s.nextLine());
+        }
+    }
+
+    /**
+     * provides a platform for users to key in tasks to add to the list
+     * @param args
```

Comment:
Should there be an empty line between the method description and the parameters? Similar to before, it would be good if the javadoc comments start with a capital letter and end with a punctuation.